kitsune/users/views.py

```diff
@@ -835,6 +817,11 @@ def get_settings(attr, *args):
             return val
         return super(FXAAuthenticateView, FXAAuthenticateView).get_settings(attr, *args)
 
+    def get(self, request):
```

Comment:
nit: You could override here the `get_extra_params` method to set the session key

Reply:
I might not be understanding, but doesn't `get_extra_params` just update the query params for the request url? Or are you saying just to update session in that method instead, e.g.
```
    def get_extra_params(self, request):
        is_contributor = request.GET.get('is_contributor') == 'True'
        request.session['is_contributor'] = is_contributor
        return self.get_settings('OIDC_AUTH_REQUEST_EXTRA_PARAMS', {})
```

Reply:
I was going for the later but let's keep it as it is.